src/pages/configurePatch/configurePatchCore/ConfigureBuildVariants.tsx

```diff
@@ -56,7 +81,7 @@ export const ConfigureBuildVariants: React.FC<Props> = ({
     }
   };
   return (
-    <StyledSiderCard>
+    <StyledSiderCard isHotKeyPressed={state.numButtonsPressed !== 0}>
```

Comment:
I get a react console warning because isHotKeyPressed gets propagated to the dom. Do ya'll know a workaround for this?

Reply:
See my comment below